### ISSUE TITLE
Add a way to force grain continuity in cutting diagram

### ISSUE THREAD

**Issue by bbeaulant** [lairdubois/lairdubois-opencutlist-sketchup-extension#371]
It would be great to be able to determine a grain continuity between multiple parts in cutting diagram.

**Comment by Kalliegro** [comment]
I am signing this petition too.
This would be a very important element of OCL.

**Comment by bbeaulant** [comment]
Yes, but the main question is how to get the desired layout .... 🤔

**Comment by Kalliegro** [comment]
1. Based on the order of the indicated elements,
2. The order of elements on the list (possible to change the position of the element on the list),
3. Based on the name of the element, e.g.: front001, front002, front003, etc.
